collateral-info.yaml

```diff
@@ -247,20 +263,34 @@ collateral-defs:
     collateral-shortname: FortunaFi DROP
     collateral-symbol: FFT1-DROP
     collateral-info-url: https://manual.makerdao.com/collateral/real-world-assets-in-detail/fortunafi
+    collateral-stability-fee-type: Fixed
+    collateral-stability-fee-payment-frequency: By the second
+    collateral-documentation: https://ipfs.io/ipfs/QmbgDoPn6UcfSDENDqHLgatMFoqXikC8E8it9WaZXyLXmc
+    collateral-geographical-exposure: US
 
   RWA007: &RWA007
     <<: *RWA-CUSTOM
     collateral-desc: Different blends of short-term bonds from ultra-short (<1 year) to short (1-3 years) maturities. Investments done through BlackRock and Baillie Gifford. Mainly invested in US Treasuries but some exposure to corporate bonds.
     collateral-longname: Monetalis Clydesdale
     collateral-shortname: Monetalis Clydesdale
     collateral-symbol: RWA007
+    collateral-stability-fee-type: Variable
+    collateral-stability-fee-payment-frequency: One-time
+    collateral-documentation: https://ipfs.io/ipfs/QmRLwB7Ty3ywSzq17GdDdwHvsZGwBg79oUTpSTJGtodToY
+    collateral-geographical-exposure: US
+    vault-closure: Upon governance request
```

Comment:
Worth adding this to the Centrifuge vaults also?

Reply:
(vault closure)

Reply:
I'm not sure how the Centrifuge deals work, but for example, the new Blocktower-Centrifuge deal, Maker cannot just unwind the position at request, the facility is planned to be 72 months long at most. so I think this is a per case basis.